src/app/app-routing.module.ts

```diff
@@ -21,6 +21,7 @@ export const routes: Routes = [
 const config: ExtraOptions = {
   useHash: false,
   relativeLinkResolution: 'legacy',
+  enableTracing: true,
```

Comment:
what does this do ?

Reply:
It's for debugging purpose, will remove that later!